### DISCUSSION TITLE
Adjusting gain for individual notes

### DISCUSSION THREAD

**Discussion by Bumblebee001** [GrandOrgue/GoOdf#59] [General]
When it came to performing this exercise I found that I had to identify pipes from a file manager as GOODF only refers to pipes in a numerical order and NOT by name i.e, Pipe001, Pipe002..... instead of 036-C, 037-C#.....

Can something be done on this issue?

**Comment by Bumblebee001** [comment]
Also..... would it be too much trouble to incorporate a simple audio player to be able to hear what effect the changes have made without having to load and re-load the sample-set in GO and deleting cache each time?

**Comment by larspalo** [comment]
Remember that you can go into the dialog for editing pipe properties and there step between the pipes in the stop/rank. In that dialog you see the name of the sample file and it's path. Otherwise, if you expand the pipe tree you can perhaps see the relative path and file name(s) too, but you might have to enlarge the program window slightly to see the full name.

> Also..... would it be too much trouble to incorporate a simple audio player to be able to hear what effect the changes have made without having to load and re-load the sample-set in GO and deleting cache each time?

I'm think that a .cmb import feature is a higher priority as all voicing then could be done in GO and when one is happy with it just transfer it to the odf via GOODF. Also, when still working on a sample set I highly recommend that you disable the automatic cache feature and not create any caches for it at all until the sample set is complete.
